src/wallet/feebumper.cpp

```diff
@@ -196,7 +196,13 @@ CFeeBumper::CFeeBumper(const CWallet *pWallet, const uint256 txidIn, const CCoin
     // moment earlier. In this case, we report an error to the user, who may use totalFee to make an adjustment.
     CFeeRate minMempoolFeeRate = mempool.GetMinFee(gArgs.GetArg("-maxmempool", DEFAULT_MAX_MEMPOOL_SIZE) * 1000000);
     if (nNewFeeRate.GetFeePerK() < minMempoolFeeRate.GetFeePerK()) {
-        vErrors.push_back(strprintf("New fee rate (%s) is less than the minimum fee rate (%s) to get into the mempool. totalFee value should to be at least %s or settxfee value should be at least %s to add transaction.", FormatMoney(nNewFeeRate.GetFeePerK()), FormatMoney(minMempoolFeeRate.GetFeePerK()), FormatMoney(minMempoolFeeRate.GetFee(maxNewTxSize)), FormatMoney(minMempoolFeeRate.GetFeePerK())));
+        vErrors.push_back(strprintf(
+            "New fee rate (%s) is lower than the minimum fee rate (%s) to get into the mempool. "
+            "The totalFee value should be at least %s or the settxfee value should be at least %s to add transaction.",
```

Comment:
Nit, there are only 2 error messages with periods, this one and:
https://github.com/bitcoin/bitcoin/blob/2c83c511dcda98550b660f6b62da7fab8548ca5c/src/wallet/feebumper.cpp#L293
How about removing the period in the last one and here: `"New fee rate ... - the totalFee value should ... to add transaction"`

Reply:
Sounds good -- done.